src/inc/quic_datapath.h

```diff
@@ -675,7 +675,7 @@ _IRQL_requires_max_(DISPATCH_LEVEL)
 _Function_class_(CXPLAT_ROUTE_RESOLUTION_CALLBACK)
 void
 (CXPLAT_ROUTE_RESOLUTION_CALLBACK)(
-    _In_ void* Context,
+    _Inout_ void* Context,
```

Comment:
Please make sure to update all implementations of this callback to use this updated SAL annotation.

Reply:
Yeah the only other one is QuicConnQueueRouteCompletion, and it was already marked Inout, so there are no more changes to be made.